kubernetes-1.31.yaml

```diff
@@ -90,6 +90,21 @@ subpackages:
           mkdir -p "${{targets.subpkgdir}}"/usr/share/bash-completion/completions
           _output/bin/kubectl completion bash > "${{targets.subpkgdir}}"/usr/share/bash-completion/completions/kubectl-${{vars.kubernetes-version}}
 
+  - name: kubectl-${{vars.kubernetes-version}}-bitnami-compat
+    description: "compat package with bitnami/kubectl image"
+    dependencies:
+      provides:
+        - kubectl-bitnami-compat=${{package.full-version}}
+      runtime:
```

Comment:
why do we need these? 🤔

Reply:
The provides? Noticed that's what we frequently do in other packages. Such as:

```
contour-1.30.yaml:        - contour-bitnami-compat=${{package.full-version}}
kafka-3.8.yaml:        - kafka-bitnami-compat=${{package.full-version}}
kubernetes-1.31.yaml:        - kubectl-bitnami-compat=${{package.full-version}}
postgresql-17-bitnami-compat.yaml:      - postgresql-bitnami-compat=${{package.full-version}}
prometheus-2.55.yaml:        - prometheus-bitnami-compat=${{package.full-version}}
rabbitmq-server-4.0.yaml:        - rabbitmq-server-bitnami-compat=${{package.full-version}}
zookeeper-3.9.yaml:        - zookeeper-bitnami-compat=${{package.full-version}}
```

Reply:
provides is correct here, yep.

Reply:
Ah, I see. `yq` is provided in `bitnami/kubectl` and bash for scripts that have bash shebang, I've seen that being used and depended on (i.e. with `set -eu -o pipefail` that requires bash).